src/test/java/com/home/eshop/dao/BikesCacheTest.java

```diff
@@ -0,0 +1,71 @@
+package com.home.eshop.dao;
+
+import com.home.eshop.model.Bike;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class BikesCacheTest {
```

Comment:
Your cache should be tested separately from other pieces of application. Use mockito as was required in a task description.

Reply:
ok